pkg/scalers/kafka_scaler.go

```diff
@@ -444,7 +444,7 @@ func (s *kafkaScaler) getConsumerOffsets(topicPartitions map[string][]int32) (*s
 func (s *kafkaScaler) getLagForPartition(topic string, partitionID int32, offsets *sarama.OffsetFetchResponse, topicPartitionOffsets map[string]map[int32]int64) (int64, int64, error) {
 	block := offsets.GetBlock(topic, partitionID)
 	if block == nil {
-		errMsg := fmt.Errorf("error finding offset block for topic %s and partition %d", topic, partitionID)
+		errMsg := fmt.Errorf("error finding offset block for topic %s and partition %d from offset block: %+v", topic, partitionID, offsets.Blocks)
```

Comment:
I think that there is a typo here: `%+v`
```suggestion
		errMsg := fmt.Errorf("error finding offset block for topic %s and partition %d from offset block: %v", topic, partitionID, offsets.Blocks)
```

Reply:
Thanks for the quick review. Yes it is my bad :sweat_smile: